libmodulemd.spec.in

```diff
@@ -132,10 +132,11 @@ mv %{buildroot}%{_bindir}/modulemd-validator \
 %doc README.md
 %if ( 0%{?rhel} && 0%{?rhel} <= 7)
 %{_bindir}/modulemd-validator2
+%{_mandir}/man1/modulemd-validator2.1*
```

Comment:
Does something the following also need to be added to the `%if` block following `%meson_install` above?
```
rename modulemd-validator modulemd-validator2 \
       %{_mandir}/man1/modulemd-validator.1*
```

Reply:
Yeah, caught that during the rewrite. Thanks.